network-api/networkapi/wagtailpages/templates/wagtailpages/fragments/blog_card.html

```diff
@@ -12,7 +12,7 @@
 each blog page's dominant tag)
 
 {% block tags %}
-  {% include "./blog_card_topics.html" with topics=page.specific.topics.all%}
+  {% include "./blog_topics.html" with topics=page.specific.topics.all%}
```

Comment:
Think there is a missing space here before the closing %

```suggestion
  {% include "./blog_topics.html" with topics=page.specific.topics.all %}
```